crates/bevy_animation/src/lib.rs

```diff
@@ -171,6 +178,13 @@ impl AnimationPlayer {
         self.elapsed = elapsed;
         self
     }
+
+    /// Handle to the animation clip.
+    ///
+    /// Can be the default Handle, which usually represents no animation.
+    pub fn animation_clip(&self) -> &Handle<AnimationClip> {
```

Comment:
Definitely feels like this should be stored as an Option instead, seeing this method.